### PR TITLE
[docs][Drivers] Modified Drivers and ORMs page for YB JDBC Driver

### PR DESCRIPTION
**Pull request by aishwarya24**
Summary:
- Modified the index page to include more details - https://deploy-preview-14414--infallible-bardeen-164bc9.netlify.app/preview/drivers-orms/java/
- Combined the content from Build an app and Drivers and ORMs for YB JDBC driver. The page includes the application code along with details on setting up the project and so on. : https://deploy-preview-14414--infallible-bardeen-164bc9.netlify.app/preview/drivers-orms/java/yugabyte-jdbc/
- Linked to the try it out reference section for examples using the smart driver.
- Removed [Build an app](https://docs.yugabyte.com/preview/develop/build-apps/java/ysql-yb-jdbc/) page.

### CODE REVIEW

File: docs/content/preview/drivers-orms/java/yugabyte-jdbc.md [hunk 1]
```diff
@@ -177,6 +196,10 @@ public class QuickStartApp {
 
 When you run the project, `QuickStartApp.java` should output something like the following:
 
+```sh
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/java/yugabyte-jdbc.md:199]: Split running and output

File: docs/content/preview/drivers-orms/java/yugabyte-jdbc.md [hunk 2]
```diff
@@ -188,7 +211,8 @@ If you receive no output or an error, check the parameters in the connection str
 
 ## Learn more
 
-- Build Java applications using [Hibernate ORM](../hibernate/)
-- [YugabyteDB JDBC driver reference](../../../reference/drivers/java/yugabyte-jdbc-reference/#load-balancing)
+- Refer to [YugabyteDB JDBC driver reference](../../../reference/drivers/java/yugabyte-jdbc-reference/#load-balancing) and [Try it out](../../../reference/drivers/java/yugabyte-jdbc-reference/#try-it-out) for detailed smart driver examples.
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/java/yugabyte-jdbc.md:214]: Don't deeplink the first link, I may just pull the load balance section now we have smart driver page

File: docs/content/preview/drivers-orms/java/yugabyte-jdbc.md [hunk 3]
```diff
@@ -188,7 +211,8 @@ If you receive no output or an error, check the parameters in the connection str
...
 - [YugabyteDB smart drivers for YSQL](../../smart-drivers/)
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/java/yugabyte-jdbc.md:215]: add spring data links as this is java

File: docs/content/preview/drivers-orms/java/_index.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ The following projects can be used to implement Java applications using the Yuga
 
 | Driver | Documentation and Guides | Latest Driver Version | Supported YugabyteDB Version |
 | ------- | ------------------------ | ------------------------ | ---------------------|
-| YugabyteDB JDBC Driver [Recommended] | [Documentation](yugabyte-jdbc/)<br />[Hello World App](../../develop/build-apps/java/ysql-yb-jdbc/)<br />[Blog](https://dev.to/yugabyte/yugabytedb-jdbc-smart-driver-for-proxyless-halb-2k8a/)<br />[Reference Page](../../reference/drivers/java/yugabyte-jdbc-reference/) | [42.3.4](https://mvnrepository.com/artifact/com.yugabyte/jdbc-yugabytedb/42.3.2/) | 2.8 and above
+| YugabyteDB JDBC Driver [Recommended] | [Documentation](yugabyte-jdbc/)<br />[Blog](https://dev.to/yugabyte/yugabytedb-jdbc-smart-driver-for-proxyless-halb-2k8a/)<br />[Reference Page](../../reference/drivers/java/yugabyte-jdbc-reference/) | [42.3.4](https://mvnrepository.com/artifact/com.yugabyte/jdbc-yugabytedb/42.3.2/) | 2.8 and above
```
Comment (ddhodge) [inline docs/content/preview/drivers-orms/java/_index.md:21]: Reference Page > Reference
everywhere; page is redundant

**Review by ddhodge — APPROVED** [review]

**Comment by aishwarya24** [comment]
Merging this and will create a new PR to address rest of the java pages based on the complexity.

**Pull request merged**